02-Context-or-Narrative/State-of-PureScript-Documentation-2019.md

```diff
@@ -17,43 +17,41 @@ It is well-known among the PureScript community that its documentation is lackin
 Outline
 - Explain what "good" documentation is and define a criteria
 - Evaluate PureScript's documentation using that criteria
-- Explain why PureScript's documentation is lacking and what is being done to improve it
+- Explore where PureScript's documentation is lacking and what can be done to improve it
 - Address various audience's possible concerns or questions centered on these themes:
     - New learners - How should I learn PureScript?
     - PureScript documentation writers - How should I write good maintainable documentation for others?
-    - Core Contributors -
+    - Core Contributors - 
 
 ## What is "Good" Documentation Anyway?
 
-Did someone ever teach you how to write "good" documentation? No, you likely just wrote what came to mind and hoped it was good enough.
-
-So, it helps to understand what is "good" documentation and why.
+Did someone ever teach you how to write "good" documentation? Probably not - you likely just wrote what came to mind and hoped it was good enough. So, it's useful to learn a definition of good documentation and understand why it's defined that way.
```

Comment:
I actually broke this up intentionally. It may not 'help the flow' as much (good point), but it makes it easier to skim. Depending on how long this document is, if we don't grab the reader's attention strongly-enough, it won't matter what we say.
Then again, I don't know if this one sentence is worth standing out in place of others...

Reply:
That makes sense. I'll review that aspect of your strategy here and post a follow-up comment. 👍

Reply:
Yeah, I think I like your approach here better. I'll revert and change from "So" to "Because of this", which I think will make it "stronger" and better able to stand on its own.

Reply:
I think `Thus` would have done the same thing ;-) but I merged it so I can move on to the next part. 

We can review this again before posting as then the rough draft of the document will be finished.